specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2016-10-01/examples/updateAccessPoliciesAdd.json

```diff
@@ -0,0 +1,44 @@
+{
+    "parameters":{
+        "resourceGroupName": "sample-group",
+        "vaultName": "sample-vault",
+        "subscriptionId":"00000000-0000-0000-0000-000000000000",
+        "api-version":"2016-10-01",
+        "parameters":{
+            "properties":{
+                "accessPolicies":[
+                    {
+                        "tenantId": "00000000-0000-0000-0000-000000000000", 
+                        "objectId": "00000000-0000-0000-0000-000000000000", 
+                        "permissions":{
+                            "keys": ["encrypt"],
+                            "secrets": ["get"],
+                            "certificates": ["get"]
+                        }
+                    }
+                ]
+            }
+        }
+     },
+    "responses": {
+        "200": {
+            "body": {
+                "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/sample-group/providers/Microsoft.KeyVault/vaults/sample-vault/accessPolicies/",
+                "type": "Microsoft.KeyVault/vaults/accessPolicies",
```

Comment:
location property missing.

Reply:
This actually was an error in the swagger def which said it returned the Vault model.  It actually returns VaultAccessPolicyParameters.  I've update the spec to reflect this.